### ISSUE TITLE
Field with `0` for data in `pie` module results in no chart

### ISSUE THREAD

**Issue by uncenter** [jaraco/svg.charts#29]
A basic usage of the `pie` module like this:
```py
from svg.charts.pie import Pie

fields = ["A", "B"]
data = [1, 13]

graph = Pie(dict(height=500, width=500, fields=fields))
graph.add_data({"data": data})

print(graph.burn())
```

Gives a nice chart like this:

![image](https://github.com/jaraco/svg.charts/assets/47499684/9dd23a2d-eea3-451a-9006-63e186663f61)

But if I change one of the fields' data to be 0:
```py
from svg.charts.pie import Pie

fields = ["A", "B"]
data = [0, 13]

graph = Pie(dict(height=500, width=500, fields=fields))
graph.add_data({"data": data})

print(graph.burn())
```

I don't get any chart, just the keys.

![image](https://github.com/jaraco/svg.charts/assets/47499684/439193ea-c1e1-47ca-ab96-d60409b3d39b)

I would expect a full circle, representing that 100% of the data is for one field, like this one from a [random pie chart generator website](https://piechartmaker.co/):

![image](https://github.com/jaraco/svg.charts/assets/47499684/2d1da1db-7aba-4ddc-90de-fd3c12b2b759)

**Comment by jaraco** [comment]
Agreed. Looks like a bug. Would you be interested in investigating and proposing a fix?

**Comment by uncenter** [comment]
Yeah I'll take a look! Not the highest of priorities at the minute but I'll investigate later this week and get back to you, maybe with a PR :)
